docs/en/ingest-management/elastic-agent/upgrade-standalone-elastic-agent.asciidoc

```diff
@@ -19,5 +19,11 @@ cd /Library/Elastic/Agent/
 sudo elastic-agent upgrade 7.10.1
 ----
 
+This command upgrades the binary. If necessary, you must update the
+configuration manually.
```

Comment:
I think a newer agent can run an older agent policy, so this should not be necessary. We could add a reminder to check if the agent policy is up to date though?